config/.eslintrc.js

```diff
@@ -84,6 +86,9 @@ module.exports = {
         '@typescript-eslint/explicit-function-return-type': 'error',
         // This check seems to be flaky, and complains about things that TS is happy about
         'react/prop-types': 'off',
+        'import/no-unresolved': 'off',
+        'indent': 'off',
+        'import/no-default-export': 'off'
```

Comment:
FYI @sndrs

Reply:
i think this is ok – requiring named exports makes it easier to trace them through codebases (same name everywhere unless you deliberately rename the import). it's fair enough if a project wants to override anything, but it proved a useful setting in frontend, prob a good default?